autoload/lsp/completion.vim

```diff
@@ -137,6 +137,57 @@ def CompletionUltiSnips(prefix: string, items: list<dict<any>>)
   endfor
 enddef
 
+# Integration with the vim-vsnip plugin
+def CompletionVsnip(items: list<dict<any>>)
+  def Pattern(abbr: string): string
+    var chars = split(escape(abbr, '\/?'), '\zs')
```

Comment:
The LSP plugin uses the method calling syntax.  Can you change this to `escape(abbr, '\/?')->split('\zs')`?